src/uct/api/version.h.in

```diff
@@ -10,9 +10,7 @@
 
 #define UCT_VERNO_MAJOR            @MAJOR_VERSION@
 #define UCT_VERNO_MINOR            @MINOR_VERSION@
-#define UCT_VERNO_REV              "@MICRO_VERSION@"
-#define UCT_VERNO_MICRO            @REV_COUNT@
-#define UCT_VERNO_STRING           "@MAJOR_VERSION@.@MINOR_VERSION@.@REV_COUNT@"
+#define UCT_VERNO_STRING           "@VERSION@"
```

Comment:
Yes, they will be different.
Currently `ucp_get_version_string()` returns string version similar to what is used in `ucp_get_version()` where there are only 3 numbers: `major`, `minor` and `patch` (previously `rev_count`). In this context it is consistent.